api/integrations/rss/rss-bot

```diff
@@ -82,9 +83,10 @@ parser.add_option('--feed-file',
                   default=os.path.join(RSS_DATA_DIR, "rss-feeds"),
                   action='store')
 parser.add_option_group(zulip.generate_option_group(parser))
-(opts, args) = parser.parse_args()
+(opts, args) = parser.parse_args() # type: Tuple[Any, Any]
```

Comment:
I think this one could be more specific - for one `args` is a `List[str]`, as per http://www.alexonlinux.com/pythons-optparse-for-human-beings.